<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>LED ON/OFF</string>
    </property>
    <widget class="QPushButton" name="btnBlue">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>99</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Blue</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>blue.png</normaloff>blue.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGreen">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>99</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Green</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>green.png</normaloff>green.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRed">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>99</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Red</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>red.png</normaloff>red.png</iconset>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOff">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>99</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>221</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>출입문 시스템</string>
    </property>
    <widget class="QPushButton" name="btnBell">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>bell.png</normaloff>bell.png</iconset>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="BtnDoor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>99</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>99</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>211</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>자동문 시스템</string>
    </property>
    <widget class="QTextEdit" name="textEdit_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>99</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStop">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>99</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>stop</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnBlue</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>blue_on()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnGreen</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>green_on()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOff</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>led_off()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRed</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>red_on()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBell</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>buzz_start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnDoor</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>servo_open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>servo_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStart</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ultra_start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ultra_stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>695</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>716</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>red_on()</slot>
  <slot>blue_on()</slot>
  <slot>green_on()</slot>
  <slot>led_off()</slot>
  <slot>buzz_start()</slot>
  <slot>servo_open()</slot>
  <slot>servo_close()</slot>
  <slot>ultra_start()</slot>
  <slot>ultra_stop()</slot>
 </slots>
</ui>
